packages/ui/src/message-pane-input/components/ToolbarBottom.jsx

```diff
@@ -82,15 +85,32 @@ const ToolbarBottom = props => {
   const handleAttachMedia = e => {
     {
       e.preventDefault();
+      // Assign files to file state
+      setFile(e.target.files[0])
       //Post request is sent here
       sendMessageHandler(attachedFile);
-
+      // Function to upload files
+      handleSubmit()
       //Then this is to clear the file from the state
       props.sentAttachedFile(null);
       clearAttached();
     }
   };
+  function handleSubmit() {
+    const url = 'http://localhost:9000/uploadFile';
```

Comment:
Put this in env file and isn't there live link for this?

Reply:
No, it doesn't actually work because the endpoints haven't been built yet but I did upload a file yesterday using an online api, I placed the API's link in the url variable

Reply:
Do you know who is to implement the backend for it?